src/localize/languages/fr.json

```
@@ -1,7 +1,20 @@
 {
   "common": {
       "version": "version",
-      "current": "Actuel"
+      "current": "Actuelle"
+  },
+  "editor": {
+    "card": {
+      "climate": {
+        "disable_window": "Desactiver fenêtre",
+        "disable_summer": "Desactiver été",
+        "disable_eco": "Desactiver mode eco",
+        "disable_heat": "Desactiver mode chauffe",
+        "disable_off": "Desactiver arret",
+        "eco_temperature": "Temperature Eco",
+        "set_current_as_main": "Echanger temperature cible avec temperature locale"
```

Shouldn't it be `Désactiver` ( https://fr.wiktionary.org/wiki/d%C3%A9sactiver ) ?
```suggestion
        "disable_window": "Désactiver fenêtre",
        "disable_summer": "Désactiver été",
        "disable_eco": "Désactiver mode eco",
        "disable_heat": "Désactiver mode chauffe",
        "disable_off": "Désactiver arrêt",
        "eco_temperature": "Température Eco",
        "set_current_as_main": "Echanger température cible avec température locale"
```